3_predictions.yml

```diff
@@ -124,8 +55,8 @@ targets:
   # get GLM reservoir predictions
   3_predictions/in/reservoir_downstream_preds.csv.ind:
     command: get_sb_item(out_ind = target_name,
-      sb_id = I('5f6a28a782ce38aaa2449137'),
-      sb_names = I('reservoir_downstream_preds.csv'))
+      sb_id = I('624f21fad34e21f82769a7b1'),
+      sb_names = I('downstream_preds.csv'))
```

Comment:
Looks like the file is still named `'reservoir_downstream_preds.csv'` on that new sb_id:
![image](https://user-images.githubusercontent.com/54007288/164315265-718bc079-99c8-421b-8f89-b2fc555bfd5c.png)

Reply:
oh yes, thanks for pointing this out. I originally had it as downstream_preds but then changed it in the data release repo to be the same as last year.